packages/quantic/quantic-examples-community/experiences/Quantic_Examples1/views/insightPanelExample.json

```diff
@@ -34,7 +34,7 @@
                 {
                   "componentAttributes": {
                     "engineId": "example-insight-panel",
-                    "insightId": "6a333202-b1e0-451e-8664-26a1f93c2faf"
+                    "insightId": "142be676-703c-445f-b2d3-fcc7c0a3ded8"
```

Comment:
why this change? did we change the config or something?

Reply:
Since we are it, let's also remove/update where this configuration id is set in other places, it is set in multiple places and it is not always useful: 

-  https://github.com/coveo/ui-kit/blob/8f4d68d7a54d6aa539ed85b7fd0516c123097eb7/packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/exampleInsightPanel.js-meta.xml#L15
- https://github.com/coveo/ui-kit/blob/8f4d68d7a54d6aa539ed85b7fd0516c123097eb7/packages/quantic/force-app/examples/main/lwc/exampleInsightPanel/exampleInsightPanel.js#L24

Reply:
Yea sure we can remove it in the default example component.

@SimonMilord This is just a different config for the HIP which has RGA enabled and is using a query we know will generate a RGA response.